crates/cli/cli/src/commands/query/clients.rs

```diff
@@ -140,13 +140,13 @@ async fn query_client_states<Chain, Counterparty>(
 ) -> Result<Vec<Client<Chain, Counterparty>>>
 where
     Chain: HasIbcChainTypes<Counterparty, ClientId = ClientId>
-        + CanQueryClientStates<Counterparty>
+        + CanQueryClientStatesWithLatestHeight<Counterparty>
```

Comment:
@soareschen I running into this error and I am at a loss as to how to fix it as I thought I had wired everything properly, but apparently not. Any idea what's the missing piece here?

```
error[E0277]: the trait bound `QueryCosmosClientStateFromAbci: ClientStatesQuerier<CosmosChain, CosmosChain>` is not satisfied
   --> crates/cli/cli/src/commands/query/clients.rs:53:45
    |
53  |         let clients = query_client_states::<CosmosChain, CosmosChain>(
    |                                             ^^^^^^^^^^^ the trait `ClientStatesQuerier<CosmosChain, CosmosChain>` is not implemented for `QueryCosmosClientStateFromAbci`
    |
    = help: the trait `ClientStatesQuerier<Chain, Counterparty>` is implemented for `QueryCosmosClientStateFromAbci`
    = note: required for `CosmosClientComponents` to implement `ClientStatesQuerier<CosmosChain, CosmosChain>`
    = note: 1 redundant requirement hidden
    = note: required for `CosmosChainComponents` to implement `ClientStatesQuerier<CosmosChain, CosmosChain>`
    = note: required for `CosmosChain` to implement `CanQueryClientStates<CosmosChain>`
    = note: required for `CosmosChain` to implement `CanQueryClientStatesWithLatestHeight<CosmosChain>`
note: required by a bound in `commands::query::clients::query_client_states`
   --> crates/cli/cli/src/commands/query/clients.rs:143:11
    |
136 | async fn query_client_states<Chain, Counterparty>(
    |          ------------------- required by a bound in this function
...
143 |         + CanQueryClientStatesWithLatestHeight<Counterparty>
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `query_client_states`
```